packages/shared/src/errors/thrower.ts

```diff
@@ -1,5 +1,6 @@
 const DefaultMessages = Object.freeze({
   InvalidFrontendApiErrorMessage: `The frontendApi passed to Clerk is invalid. You can get your Frontend API key at https://dashboard.clerk.dev/last-active?path=api-keys. (key={{key}})`,
+  InvalidProxyUrlErrorMessage: `The proxyUrl passed to Clerk is invalid. Proxy url should be an absolute URL or a relative path  with a leading '/'. (key={{url}})`,
```

Comment:
:wrench: We should either use the uppercase word or lowercase for url and not mix "url" and "URL" in the same sentence.

Reply:
```suggestion
  InvalidProxyUrlErrorMessage: `The proxyUrl passed to Clerk is invalid. The expected value for proxyUrl is an absolute URL or a relative path with a leading '/'. (key={{url}})`,
```